pilot/pkg/controller/workloadentry/workloadentry_controller.go

```diff
@@ -675,3 +763,46 @@ func (c *Controller) handleErr(err error, key interface{}) {
 func makeProxyKey(proxy *model.Proxy) string {
 	return string(proxy.Metadata.Network) + proxy.IPAddresses[0]
 }
+
+func lookupWorkloadEntryByProxyID(store model.ConfigStoreCache, proxy *model.Proxy) string {
+	if len(proxy.IPAddresses) == 0 {
+		return ""
+	}
+
+	parts := strings.Split(proxy.ID, ".")
+	if len(parts) != 2 || parts[1] != proxy.Metadata.Namespace {
```

Comment:
can use `proxy.ConfigNamespace` instead of parts[1]

Reply:
Source code in this file uses `proxy.Metadata.Namespace` everywhere.

I can change to `proxy.ConfigNamespace`.

Should I change only this occurrence, or all occurrences in this file?

Reply:
this function has been removed